src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/operations/streaming_policy.py

```diff
@@ -90,12 +155,12 @@ def _parse_key_to_track_mappings_json(key_to_track_mappings):
                     str_policy_content_key = StreamingPolicyContentKey(**str_policy_content_key_json)
                     key_to_track_mappings_result.append(str_policy_content_key)
             except:
-                raise CLIError('Malformed key-to-track-mappings JSON argument. Please, make sure you are sending a list of TrackSelection.' \
-                    ' For further information, please refer to https://docs.microsoft.com/en-us/rest/api/media/streamingpolicies/create#trackselection')
+                raise CLIError(errorMessage)  # pylint: disable=line-too-long
     return key_to_track_mappings_result
 
 
 def _parse_clear_tracks_json(clear_tracks):
+    errorMessage = 'Malformed key-to-track-mappings JSON argument. Please, make sure you are sending a list of TrackSelection. For further information, please refer to https://docs.microsoft.com/en-us/rest/api/media/streamingpolicies/create#trackselection'  # pylint: disable=line-too-long
```

Comment:
Repeating the definition twice. Why not maybe inject it from where they call from?

Reply:
Because both methods get called in different ocations and this would mean having to define the message error multiple times in the 3 factory methods, this solution is the best one I found

Reply:
after the change you are sending an errorMessage to the function. then you should move the definition of errorMessage to wherever you invoke it